cassandra-examples/README.adoc

```diff
@@ -0,0 +1,13 @@
+= Vert.x Cassandra Client examples
+
+== Start Cassandra instance
+
+In order to run this examples you need a cassandra instance running locally.
+The easiest way to do this is using link:https://github.com/riptano/ccm[ccm]:
```

Comment:
Instead of saying _the easiest way_ you could coopy the installation procedure explained on https://github.com/riptano/ccm#installation

The users will figure out the details for each Linux distro or specific pip install.

Reply:
I would not copy, I would refer.